docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,125 @@
+---
+product: material-ui
+title: React Modal component
+components: Modal, ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
+---
+
+# Modal
+
+<p class="description">The `Modal` component lets you create dialogs, popovers, lightboxes, and other windows that force the user to take action before continuing.</p>
```

Comment:
```suggestion
<p class="description">The `Modal` component lets you create dialogs, popovers, lightboxes, and other elements that force the user to take action before continuing.</p>
```
_Windows_ is just sounding weird in this context, _element_ sounds better for some reason... Not a strong opinion, though.

Reply:
I opted for "window" since that's the terminology used in the Wikipedia entry, but I'm not married to it either.